src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/Areas/Identity/RevalidatingAuthenticationStateProvider.cs

```diff
@@ -0,0 +1,98 @@
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace RazorComponentsWeb_CSharp.Areas.Identity
+{
+    /// <summary>
+    /// An <see cref="AuthenticationStateProvider"/> service that revalidates the
+    /// authentication state at regular intervals. If a signed-in user's security
+    /// stamp changes, this revalidation mechanism will sign the user out.
+    /// </summary>
+    /// <typeparam name="TUser">The type encapsulating a user.</typeparam>
```

Comment:
Should we have a comment here that says something like: "this is an important security feature, do not modify it unless you have a good reason"?

Reply:
I don't know, that seems pretty strange. Certain modifications would be totally reasonable (e.g., changing `RevalidationInterval` to a shorter or longer duration).

If you feel strongly, and want to suggest a particular phrasing, please let me know. Otherwise it seems more normal in our templates not to add scare-comments. For example we don't have similar comments about `app.UseHsts` or `app.UseHttpsRedirection`.